swerve100/src/main/java/org/team100/lib/motor/turning/FalconTurningMotor.java

```diff
@@ -22,23 +24,54 @@ public static class Config {
     public FalconTurningMotor(String name, int canId) {
         m_motor = new WPI_TalonFX(canId);
         m_motor.configFactoryDefault();
+        m_motor.configSelectedFeedbackSensor(FeedbackDevice.IntegratedSensor);
         m_motor.setNeutralMode(NeutralMode.Brake);
         m_motor.setInverted(InvertType.InvertMotorOutput);
         m_motor.configStatorCurrentLimit(
                 new StatorCurrentLimitConfiguration(true, m_config.kCurrentLimit, m_config.kCurrentLimit, 0));
         m_motor.configSupplyCurrentLimit(
                 new SupplyCurrentLimitConfiguration(true, m_config.kCurrentLimit, m_config.kCurrentLimit, 0));
         SmartDashboard.putData(String.format("Falcon Turning Motor %s", name), this);
+        m_motor.configNominalOutputForward(0);
+        m_motor.configNominalOutputReverse(0);
+        m_motor.config_kF(0, 0);
+        m_motor.config_kP(0, 0.05);
+        m_motor.config_kI(0, 0);
+        m_motor.config_kD(0, 0);
+        m_motor.configVoltageCompSaturation(11);
+        m_motor.enableVoltageCompensation(true);
     }
 
     @Override
     public double get() {
         return m_motor.get();
     }
 
-    //THIS DOES NOT ACTUALLY SET PID This is just here for the CAN turning motor
-    public void setPID(ControlMode control, double output) {
-        this.set(output);
+    public void setPIDVelocity(double outputRadiansPerSec, double outputRadiansPerSecPerSec) {
+        double gearRatio = 10.29;
+        double ticksPerRevolution = 2048;
+        double revolutionsPerSec = outputRadiansPerSec/(2*Math.PI);
+        double revolutionsPerSec2 = 0;
+        double revsPer100ms = revolutionsPerSec / 10;
+        double ticksPer100ms = revsPer100ms * ticksPerRevolution;
+        DemandType type = DemandType.ArbitraryFeedForward;
+        double Kn = 0.75/6.6*gearRatio;//1.296
+        double Kf = 0.6;//0.3145
+        double Ke = 0.0688420763;
+        double Ks = 0.05/6.6*gearRatio;
+        double VSat = 11;
+        if (revolutionsPerSec<.25) {
```

Comment:
this is where you'd want to use the *actual* wheel speed, not the commanded speed, right?